theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-sub-widgets/trace-explorer-keyboard-shortcuts/zoom-pan-shortcuts-table.tsx

```diff
@@ -58,6 +58,20 @@ export class ZoomPanShortcutsTable extends React.Component {
                 <div className='shortcuts-table-column'>
                     <table>
                         <tbody>
+                            <tr>
+                                <td><i className='fa fa-undo fa-lg' /> Undo</td>
+                                <td className='monaco-keybinding shortcuts-table-keybinding'>
+                                    <span className='monaco-keybinding-key'>CTRL</span>
+                                    <span className='monaco-keybinding-key'>Z</span>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td><i className='fa fa-repeat fa-lg' /> Redo</td>
+                                <td className='monaco-keybinding shortcuts-table-keybinding'>
+                                    <span className='monaco-keybinding-key'>CTRL</span>
+                                    <span className='monaco-keybinding-key'>Y</span>
+                                </td>
+                            </tr>
```

Comment:
Does this bind the command to the main menu `Edit->Undo` and `Edit->Redo`. It would be good if the main menu is also triggering the command if the trace viewer is in focus.

To be consistent the redo should CTRL+SHIFT+Z.